package.json

```diff
@@ -12,8 +12,11 @@
     "start": "NODE_ENV=production node build/server.js",
     "speculate": "speculate",
     "storybook": "start-storybook -p 9001 -c .storybook",
-    "test:e2e": "npx cypress run",
-    "test:e2e:interactive": "npx cypress open",
+    "cypress": "npx cypress run",
+    "cypress:interactive": "npx cypress open",
+    "test:e2e": "(lsof -t -i:7080 | xargs kill) && npm run build && run-p --race start cypress",
+    "test:e2e:ci": "npm run build && run-p --race start cypress",
```

Comment:
I appreciate that we don't expect an app to be listening to 7080 on CI but is it a problem if we do kill it?

Wondering whether we could remove this line for brevity.

Reply:
On CI we shouldn't have anything running on the port. I'd rather not run a kill command on CI if not necessary.